mongofiles/mongofiles.go

```diff
@@ -111,7 +115,19 @@ func (mf *MongoFiles) ValidateCommand(args []string) error {
 		} else {
 			mf.FileName = args[1]
 		}
-	case Search, Put, Get, Delete:
+	case Put, Get:
+		// monogofiles --put and mongofiles --get should work over
+		// a list of files, i.e. by using mf.FileNameList -- see
+		// TOOLS-2667 for user-level specification
+		if len(args) == 1 || args[1] == "" {
+			return fmt.Errorf("'%v' argument missing", args[0])
+		}
+
+		if len(args) == 2 {
+			mf.FileName = args[1]
```

Comment:
This doesn't seem necessary; can't `--put` and `--get` with one argument still be handled the same way as with many arguments? The list just has one filename in that case but that's ok.

Reply:
I can remove it -- I don't think it should break anything.

Reply:
Do you still want me to do this, even though I'd have to move the existing tests around?

Reply:
Yes, I still think this is worth doing. That test is checking that `put foo` creates the expected `MongoFiles` and I'm proposing we change what the expected `MongoFiles` looks like. This will entail changing at least that test (and possibly others).